twindb_backup/modifiers/base.py

```diff
@@ -0,0 +1,41 @@
+from contextlib import contextmanager
+
+
+class ModifierException(Exception):
+    """Base Exception for Modifier error"""
+
+
+class Modifier(object):
```

Comment:
I don't think we should allow the base class to be instantiated directly, rather only through inheritance.